recipes/r-scbio/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "0.1.4" %}
+
+package:
+  name: r-scbio
+  version: {{ version }}
+
+source:
+  url:
+    - https://github.com/amitfrish/scBio/releases/download/{{ version }}/scBio_{{ version }}.tar.gz
```

Comment:
This URL is probably more stable as it is automatically generated by github: 
```
https://github.com/amitfrish/scBio/archive/refs/tags/{{ version }}.tar.gz
```

The other archive is manually generated and uploaded by the author.